brd-ios/breadwallet/src/Redesign/Scenes/Buy/CardSelection/CardSelectionViewController.swift

```diff
@@ -66,4 +70,20 @@ class CardSelectionViewController: ItemSelectionViewController {
         
         return cell
     }
+    
+    // MARK: - Additional Helpers
+    func showActionSheetRemovePayment(instrumentID: String) {
+        let optionMenu = UIAlertController(title: nil, message: nil, preferredStyle: .actionSheet)
```

Comment:
presenter should prepare the VM's / configs for the alert view.. and coordinator displays it :)